### PR TITLE
refresh token

### PR DESCRIPTION
**Pull request by AntonPuko**
close https://github.com/Brewskey/Brewskey.App/issues/306

### CODE REVIEW

File: src/stores/AuthStore.js [hunk 1]
```diff
@@ -84,17 +82,33 @@ class AuthStore {
...
-      this.isReady = true;
-    });
+
+      runInAction(() => {
+        if (newAuthState && newAuthState.accessToken) {
+          this._authState = newAuthState;
+        }
+      });
+    } catch (error) {
+      // swallow
```
Comment (jlkalberer) [inline src/stores/AuthStore.js]: This seems like a bad idea

Comment (AntonPuko) [reply src/stores/AuthStore.js]: when we get an error here, it drops to login page (by reaction triggered by finally case)..I don't think that user should know any reason why error appears during refresh token updating.

Comment (jlkalberer) [reply src/stores/AuthStore.js]: Yeah, but we should handle it better. If there is an error with the refresh token, it probably means that the auth token is expired too.

Comment (AntonPuko) [reply src/stores/AuthStore.js]: ok, I'll remove authstate from local storage in catch block

**Pull request merged**